docs/tutorials/Soil/richards_equation.jl

```diff
@@ -147,16 +150,35 @@ Y.soil.ϑ_l .= FT(0.494);
 # Next, we turn to timestepping. We choose the initial and final times, as well as a timestep.
```

Comment:
# As usual, your timestep depends on the problem you are solving, the accuracy of the solution required, and the timestepping algorithm you are using.